webapp/src/main/webapp/i18n/vivo_all.properties

```diff
@@ -810,3 +810,6 @@ role_in_presentation_capitalized=Role in Presentation
 advisee_capitalized_first_name=First Name
 advisee_capitalized_lastname=Last Name
 
+label.addRoleToPersonTwoStage.start_capitalized = Start
+label.addRoleToPersonTwoStage.end_capitalized = End
```

Comment:
We should also see updates in the i18n files, no?
https://github.com/vivo-project/VIVO-languages/tree/master/webapp/src/main/webapp/i18n

Reply:
ok done with: https://github.com/vivo-project/VIVO-languages/pull/19